index.bs

```diff
@@ -155,10 +155,10 @@ User agents must also provide an event handler IDL attribute [[!HTML]] named {{W
 
 <pre class="idl">
 partial interface Window {
-    attribute EventHandler ondeviceorientation;
+    [SecureContext] attribute EventHandler ondeviceorientation;
```

Comment:
Add this attribute to `ondeviceorientationabsolute` as well.

Reply:
Done.